### ISSUE TITLE
pvt regions for a single well

### ISSUE THREAD

**Issue by GitPaean** [OPM/opm-simulators#1729]
It looks we use the PVT region in the first (maybe not top as indicated in the following code) perforated cells as the PVT region of the well. 

https://github.com/OPM/opm-simulators/blob/ded12a60597c3d624c0fca643ae40f6e3d7e6350/opm/autodiff/BlackoilWellModel_impl.hpp#L548

while in WELSPECS, item 11, it looks like we can either specify it, or by default (0), we use the PVT region of the lowest perforation as the PVT region for the well, following the OPM manual. 

Do I understand this part wrongly? any comment, @totto82 ?

And also for the PVT region in FluidState, the pvt region begins with 0 instead of 1, right?  I mean, if deck specifies 1, in the FluidState, the pvt region will be 0?

**Comment by totto82** [comment]
Currently we dont support setting the pvt region in WELSPECS. I suggest to add a pvt_region_idx_ member for each well that are set in the initialization either by WELSPECS or by picking the pvt_region from one of the perforation. Maybe even adding a test that prints a warning if the well perforate in different pvt regions. A well that perforates more than one region is definatly not recomenteded.

**Comment by GitPaean** [comment]
Yes, but when we do not set a PVT region for the well, there will be a default PVT region. Currently, we use the pvt region of the first perforated cell, while the more correct one is to use the lowest perforated cell following the manual.
